ui/apps/dev-dashboard/src/components/onboarding/generate-message.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+import Text from '@akashaorg/design-system-core/lib/components/Text';
+import {
+  SteppedActionWrapper,
+  SteppedActionWrapperProps,
+} from '@akashaorg/design-system-components/lib/components/SteppedActionWrapper';
+
+import { DevMessageForm, DevMessageFormProps } from '../common';
+
+export type GenerateMessageProps = DevMessageFormProps &
+  SteppedActionWrapperProps & {
+    ctaIntroLabel?: string[];
+    onCTAClick?: () => void;
+  };
+
+export const GenerateMessage: React.FC<GenerateMessageProps> = props => {
+  const { ctaIntroLabel, onCTAClick } = props;
+
+  const handleCTAClick = () => {
+    if (onCTAClick && typeof onCTAClick === 'function') onCTAClick();
+  };
+
+  return (
+    <SteppedActionWrapper {...props}>
+      {ctaIntroLabel && ctaIntroLabel.length && (
+        <Text>
+          {ctaIntroLabel[0]}{' '}
```

Comment:
you can spread this array to have a more meaningful naming:
```
const [textDescription, buttonLabel, otherDescription] = ctaIntroLabel;
```
replace var names in array with proper naming

Reply:
is this solved? I don't see any changes...